app/helpers/foreman_openscap_helper.rb

```diff
@@ -15,10 +15,15 @@ def scap_doc_url(section = '')
     documentation_url(section, root_url: scap_root_url)
   end
 
+  def doc_flavor
+    ForemanOpenscap.with_katello? ? 'katello' : 'foreman-el'
```

Comment:
My biggest concern is that the OpenSCAP guide is not published in stable branches as `foreman-el`:

https://github.com/theforeman/foreman-documentation/blob/5a811af4a30310b96a15e3bec708079e04256c31/guides/doc-Managing_Security_Compliance/master.adoc#L14-L18

This results in linking to https://docs.theforeman.org/3.8/Managing_Security_Compliance/index-foreman-el.html (for Foreman 3.8).

This is because it hasn't been properly reviewed. You can see examples such as https://docs.theforeman.org/nightly/Managing_Security_Compliance/index-foreman-el.html#Deploying_a_Policy_in_a_Host_Group_Using_Ansible_security-compliance

> You have enabled and synced the operating system repositories to Foreman, and enabled them on the hosts:

Content syncing is not available without Foreman, so that needs to be addressed.

We do publish the nightly docs, just so you can at least review some of it, even if it's broken.